messages/password.generate.json

```diff
@@ -8,7 +8,11 @@
   "flags": {
     "onBehalfOf": "comma-separated list of usernames or aliases to assign the password to"
   },
-  "noSelfSetError": "Create a scratch org with the enableSetPasswordInApi org security setting set to TRUE and try again.",
+  "noSelfSetErrorV50": "Create a scratch org with the enableSetPasswordInApi org security setting set to TRUE and try again.",
+
+  "noSelfSetError": "Beginning in version 51, enableSetPasswordInApi is a feature in your scratch org definition file.  Settings.securitySettings.passwordPolicies.enableSetPasswordInApi is deprecated.",
+  "noSelfSetErrorAction": "Update your scratch org definition file and try creating the org again",
```

Comment:
Suggestion (I think you can add multiple lines to an action, it would be nice for it to look something like this): 

-----start edit-----

Update your scratch org definition file and remove the "securitySettings" setting. Then add EnableSetPasswordInApi as a feature.  For example:

"features": ["EnableSetPasswordInApi"],

Then try creating the scratch org again.